classes.js

```diff
@@ -0,0 +1,259 @@
+let imgSrc = [
+    "images/player/face.png",
+    "images/player/profileboy.png",
+    "images/player/profilegirl.png"
+];
+
+let imgArr = [];
+let imgFace = new Image()
+imgFace.src = imgSrc[0]
+imgArr.push(imgFace)
+let imgLeft = new Image()
+imgLeft.src = imgSrc[2]
+imgArr.push(imgLeft)
+let imgRight = new Image()
+imgRight.src = imgSrc[1]
+imgArr.push(imgRight)
+
+
+let player = {
+    x: canvas.width / 2 - 100,
+    y: 400,
+    width: 200,
+    height: 220,
+    img: new Image(),
+    // to move the player to the right
+    rightPressed: function () {
+        // to keep it inside the canvas
+        if (this.x < canvas.width - this.width + 40) {
+            this.x += 50;
+        }
+    },
+    // to move the player to the left
+    leftPressed: function () {
+        // to keep it inside the canvas
+        if (this.x >= 0) {
+            this.x -= 50;
+        }
+    },
+    upPressed: function () {
+        if (this.y >= 5) {
+            this.y -= 40;
+        }
+    },
+    downPressed: function () {
+        if (this.y <= canvas.height - this.height) {
+            this.y += 40;
+        }
+    },
+    // to draw the player on the canvas
+    update: function () {
+        ctx.fillStyle = "red";
+
+
+        if (gameStart) {
+            this.img = imgArr[0];
+            gameStart = false;
+        }
+        if (UpDownDir) {
+            this.img = imgArr[0];
+            UpDownDir = false;
+        }
+        if (leftDir) {
+            this.img = imgArr[1];
+            leftDir = false;
+        }
+        if (rightDir) {
+            this.img = imgArr[2];
+            rightDir = false;
+        }
+        ctx.drawImage(this.img, this.x, this.y, this.width, this.height);
+    },
+    // declare the player borders
+    left: function () {
+        return this.x + 30;
+    },
+    top: function () {
+        return this.y + 30;
+    },
+    right: function () {
+        return this.x + this.width - 30;
+    },
+    bottom: function () {
+        return this.y + this.height - 30;
+    },
+    // to check if the player collide with the objects
+    crash: function (object) {
+        return !(
+            this.bottom() < object.top() ||
+            this.top() > object.bottom() ||
+            this.right() < object.left() ||
+            this.left() > object.right()
+        );
+    }
+};
+// Create object class
+class object {
```

Comment:
even though possible I would not call a class "object" that could be very confusing .. maybe "Obstacle" or "Collectible" or something